### DISCUSSION TITLE
How to exposes and import a .ts file ?

### DISCUSSION THREAD

**Discussion by NicoJolyAdeo** [module-federation/module-federation-examples#2770] [Q&A]
Hi,
In Vue 3 i create a test.ts file (a composables) 
`
export const getTest = () => {
  return "123";
};
`
and want to exposes this file :
`
name: "my_microfront",
        filename: "remoteEntry.js",
        exposes: {
          "./test": "./src/composables/test.ts",
        },
`

And now want to import in host :
`
import { getTest } from "my_microfront/test";
`
But i have this error :
TS2614: Module '"*.vue"' has no exported member 'getTest'. Did you mean to use 'import getTest from "*.vue"' instead?

**Comment by NicoJolyAdeo** [comment]
Is it a bad question ?

**Reply by belinef** [reply]
@NicoJolyAdeo i don't have any issues 
please see.
https://github.com/module-federation/module-federation-examples/pull/2786/files
